scripts/compose.py

```diff
@@ -1051,6 +1050,17 @@ def __init__(self, **options):
                 self.environment["ELASTICSEARCH_PASSWORD"] = "changeme"
                 self.environment["ELASTICSEARCH_USERNAME"] = "kibana_system_user"
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
+        self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
+            "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+
+    @classmethod
+    def add_arguments(cls, parser):
+        parser.add_argument(
+            "--kibana-elasticsearch-url",
+            action="append",
+            dest="kibana_elasticsearch_urls",
```

Comment:
What's the reason for using an argument as singular-based and a variable as plural-based?

Reply:
kibana_elasticsearch_urls is an array the URLs